### ISSUE TITLE
Proxy Class Generator: remove the date from the extension files

### ISSUE THREAD

**Issue by terabyte-gc** [codingjungle/toolbox#42]
**Is your feature request related to a problem? Please describe.**
The files inside the folder **\dtProxy\extensions** all have a `@since		DD MMM YYYY` value in the header. This value changes every time the tool is run (as long as it's a different day) and all files get flagged as modified.

**Describe the solution you'd like**
I edited the file "\applications\toolbox\sources\Proxy\Generator\Extensions.php" and changed this code (lines 74-80):
```
                        $replace = [
                            $key->directory,
                            date('d M Y'),
                            $key->directory,
                            $file->getBasename('.txt'),
                            '',
                        ];
```
with this:
```
                        $replace = [
                            $key->directory,
                            #date('d M Y'),
                            '',
                            $key->directory,
                            $file->getBasename('.txt'),
                            '',
                        ];
```

Basically I just commented the date() replacement line and replaced it with an empty string so the value doesn't change.

Another option would be to simply not find the `{date}` value at all in the first place. As long as the file is not flagged as modified because the date changes the solution doesn't really matter.

**Comment by codingjungle** [comment]
I've always considered the proxyclasses to be transitory, but this fix should harm or effect anything, was just replace the placeholders IPS have in their txt files anyway for "completeness", i suppose the only one i really need there is the removal of the php opening tag as the file writer adds that already.
